src/runtime_src/core/pcie/linux/shim.cpp

```diff
@@ -76,6 +76,9 @@
 
 #define SHIM_QDMA_AIO_EVT_MAX   1024 * 64
 
+//#include "core/pcie/driver/linux/xocl/userpf/common.h"
+//MAX_DEPs copied from above include file.
+#define MAX_DEPS        8
```

Comment:
can we put MAX_DEPS in a header file that is common for both driver and user space code?